bower.json

```diff
@@ -36,7 +36,8 @@
     "angular-xeditable": "^0.2.0",
     "angular-charts": "^0.2.7",
     "d3": "^4.2.2",
-    "semantic": "semantic-ui#^2.2.2"
+    "semantic": "semantic-ui#^2.2.2",
+    "angular-ui-ace": "bower"
```

Comment:
I haven't seen this before, what does it mean to have `"bower"` as the version value?